vmm/src/vmm_config/vsock.rs

```diff
@@ -13,6 +13,8 @@ pub struct VsockDeviceConfig {
     pub id: String,
     /// A 32-bit Context Identifier (CID) used to identify the guest.
     pub guest_cid: u32,
+    /// Path to local unix socket.
+    pub uds_path: String,
```

Comment:
This should be a `VsockBackend` instead.

Reply:
I agree. However, currently, there's no clean way of handling these config structs properly. The time is close when we'll have to, though, and this will get solved.